components/schedule-editor/README.md

```diff
@@ -26,22 +26,22 @@ Nylas Schedule Editor is currently in active development. Want to contribute? [F
 
 ## Documentation
 
-- [Nylas Scheduler reference](https://developer.nylas.com/docs/user-experience/components/scheduler-editor-component/)
+- [Nylas Booking reference](https://developer.nylas.com/docs/user-experience/components/scheduler-editor-component/)
 
-- [Examples](https://components.nylas.io/components/scheduler-editor/src/index.html)
+- [Examples](https://components.nylas.io/components/schedule-editor/src/index.html)
```

Comment:
Lots of small changes like this one; the original paths and links were typo'd as `scheduler-editor` instead of `schedule-editor`.